api/src/main/java/io/strimzi/api/kafka/model/KafkaUserQuotas.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import io.strimzi.crdgenerator.annotations.Description;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Collections.emptyMap;
+
+/**
+ * Represent the Quotas configuration for Kafka User
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode
+public class KafkaUserQuotas implements UnknownPropertyPreserving, Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private Integer producerByteRate;
+    private Integer consumerByteRate;
+    private Integer requestPercentage;
+
+    private Map<String, Object> additionalProperties;
+
+    @Description("A quota on the maximum bytes per-second that each client group can publish to a broker before the clients in the group are throttled. Defined on a per-broker basis.")
+    @Minimum(1)
+    public Integer getProducerByteRate() {
+        return producerByteRate;
+    }
+
+    public void setProducerByteRate(Integer producerByteRate) {
+        this.producerByteRate = producerByteRate;
+    }
+
+    @Description("A quota on the maximum bytes per-second that each client group can fetch from a broker before the clients in the group are throttled. Defined on a per-broker basis.")
+    public Integer getConsumerByteRate() {
+        return consumerByteRate;
+    }
+
+    public void setConsumerByteRate(Integer consumerByteRate) {
+        this.consumerByteRate = consumerByteRate;
+    }
+
+    @Description("A quota on the maximum CPU utilization of each client group as a percentage of network and I/O threads.")
+    @Minimum(0)
```

Comment:
What does 0% do? It seems strange that you:
* Use 1 byte for producerByteRate
* Set nothing for consumerByteRate
* Set 0 for requestPercentage

I wonder if that has some reason or whether we should make it more consistent.

Reply:
Yeah, I think for `consumerByteRate` it should be minimum 1 as well. These values are the output of the previous conversation with @tombentley

Reply:
So why is minimum byte rate 1 but request percentage 0?

Reply:
I guess all three can be minumum 0.